tests/python/pytorch/graphbolt/impl/test_fused_csc_sampling_graph.py

```diff
@@ -2219,10 +2219,13 @@ def test_sample_neighbors_hetero_pick_number(
         type_per_edge=type_per_edge,
         node_type_to_id=ntypes,
         edge_type_to_id=etypes,
-    )
+    ).to(F.ctx())
 
     # Generate subgraph via sample neighbors.
-    nodes = torch.LongTensor([0, 1])
+    nodes = {
```

Comment:
Is this change needed so that seed_offsets can be generated before sample neighbors is called?

Reply:
I wonder why the GPU tests passed without this change.

Reply:
Yes it's needed to generate seed_offsets. I guess this test wasn't run on GPU entirely because the graph wasn't carried to GPU so it didn't go into the CUDA branch.

Reply:
Good catch.